mogp_emulator/GaussianProcessGPU.py

```diff
@@ -476,7 +594,7 @@ def predict(self, testing, unc=True, deriv=True, include_nugget=True):
         :func:`mogp_emulator.GaussianProcess.GaussianProcess.predict`
         """
 
-        if self.theta is None:
+        if not self.theta.data_has_been_set():
```

Comment:
yep, `gp.theta` is pointing to the (python binding of  the) C++ object, and for some reason I did find it useful to have this flag (implemented in `gpparams.hpp` - I think because it would otherwise have been hard to distinguish the case where the C++ params object had been instantiated but no parameters set, and where it didn't exist at all...

Reply:
Ah, makes sense.